checkov/terraform/checks/terraform/base_registry.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict, Any, Tuple
+
+from checkov.common.checks.base_check_registry import BaseCheckRegistry
+
+
+class Registry(BaseCheckRegistry):
+    def extract_entity_details(self, entity: Dict[str, Any]) -> Tuple[str, str, Dict[str, Any]]:
+        terraform_configuration = dict(entity.items())
+
+        if '__startline__' not in terraform_configuration or '__endline__' not in terraform_configuration:
```

Comment:
Please import `START_LINE` and `END_LINE` from `checkov.common.utils` instead of using the const :)